### DISCUSSION TITLE
Consulta TP3 - Incidentes - Falla de Conexión

### DISCUSSION THREAD

**Discussion by matiastito** [dds-jv/foro#438] [TPA - 2024]
Buenas a todos, espero esten bien.

El motivo por el cual los molestamos es porque estamos modelando los Incidentes. No nos queda claro el siguiente requerimiento:

> Fallas de Conexión:
> - si se deja de recibir la temperatura de la heladera: si no se reciben lecturas por mas de 15mínutos, se considera que hay una falla de coexión.

Encontramos acá una dificultad para modelar esos 15minutos. La verdad es que no se nos ocurrió mucho porque si bien **no tenemos inconvenientes al momento de registrar la úlitma lectura**, eso lo podemos hacer faćil, no encontramos la mejor manera de modelar ese cheueo cada _15 minutos_:
- ¿Es un 'timmer' que debe tener la heladera?.
- O podemos tener una tarea programada que corra cada 15minutos, itere todas las heladeras chequeando la úlitma lecutra.

En fin, eso por ahora...

Muchas gracias y esperamos haber sido claros en la consulta.

Saludos!

**Comment by flbulgarelli** [comment]
> ¿Es un 'timmer' que debe tener la heladera?.

El chequeo debería ser del lado de nuestro sistema, no de la heladera, dado que eso es justamente lo que estamos implementando.

> * O podemos tener una tarea programada que corra cada 15minutos, itere todas las heladeras chequeando la úlitma lecutra.

¡Esto tiene mucho sentido!
